runtime/compiler/runtime/RelocationRecord.cpp

```diff
@@ -2277,27 +2293,22 @@ TR_RelocationRecordInlinedMethod::inlinedSiteValid(TR_RelocationRuntime *reloRun
             }
          else
             {
-            inlinedSiteIsValid = false;
```

Comment:
Is there any particular significance in moving this line below the assertion? Just curious; it looks harmless

Reply:
Initially, I had moved the code that checks `compileRomClass == currentRomClass` into `inlinedSiteCanBeActivated` at whcih point the `inlinedSiteIsValid = false;` got changed to a `return false`, and so had to be moved after the check for SVM. It turns out commoning that code to also be used for profiled inlined methods didn't work (because the rom class stored in the relorecord was not the rom class of the method that was inlined), so I had to un-refactor the code. I guess that resulted in the `inlinedSiteIsValid = false` ended up after the if :).